bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/tt0123j_1l_tbar_5f_ckm_LO_MLM/tt0123j_1l_tbar_5f_ckm_LO_MLM_madspin_card.dat

```diff
@@ -0,0 +1,34 @@
+#************************************************************
+#*                        MadSpin                           *
+#*                                                          *
+#*    P. Artoisenet, R. Frederix, R. Rietkerk, O. Mattelaer * 
+#*                                                          *
+#*    Part of the MadGraph5_aMC@NLO Framework:              *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#initialization parameters
+ set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+# set BW_cut 15                # cut on how far the particle can be off-shell
```

Comment:
please remove commented lines

Reply:
Hi Marco, you mean remove just 

set BW_cut 15

Or all lines 16-18?

Also should this be done for all the new cards or just the tt1023l_1l_tbar_5f_ckm_LO_MLM sample?

Reply:
only line 18